app/models/rule_praxi.rb

```diff
@@ -0,0 +1,24 @@
+class RulePraxi < ApplicationRecord
```

Comment:
so, praxis is the actual name, it's not a plural of praxi

```suggestion
class RulePraxis < ApplicationRecord
```

Reply:
I think this should be following the RoR inflection pattern now with the `RulePraxis` model name